demo/screens/EthereumQuestion.js

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { Image, Button, StyleSheet, View, Text } from "react-native";
+import { createStackNavigator, createAppContainer } from "react-navigation";
+
+export default class EthereumQuestion extends React.Component {
+  render() {
+    return (
+      <View style={styles.container}>
+        <Text style={styles.questionText}>{`Are you new to Ethereum?`}</Text>
+        <View style={styles.buttonView}>
+          <Button
+            title="Yep"
+            onPress={() => this.props.navigation.navigate("EthereumSignUp")}
+          />
+        </View>
+        <View style={styles.buttonView}>
+          <Button
+            title="Nope"
+            onPress={() => this.props.navigation.navigate("EthereumSignIn")}
+          />
+        </View>
+      </View>
+    );
+  }
+}
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    backgroundColor: "#fff",
```

Comment:
This can live in a constants file we import from

Reply:
Ok, sure.